src/librustc_mir/dataflow/framework/cursor.rs

```diff
@@ -72,209 +73,134 @@ where
         self.state.contains(elem)
     }
 
-    /// Resets the cursor to the start of the given basic block.
-    pub fn seek_to_block_start(&mut self, block: BasicBlock) {
-        self.state.overwrite(&self.results.borrow().entry_sets[block]);
-        self.pos = CursorPosition::BlockStart(block);
-        self.success_effect_applied = false;
-    }
-
-    /// Advances the cursor to hold all effects up to and including to the "before" effect of the
-    /// statement (or terminator) at the given location.
+    /// Resets the cursor to hold the dataflow state for the given basic block at fixpoint.
```

Comment:
likewise, here, you probably want to make sure this comment gets revised.

(especially since this sentence *does* have meaning, under my interpretation of "fixpoint", since the phrase "at fixpoint" could denote "the dataflow state at the (least) fixed point"; or, it could, as I suspect, carry your denotation of the so-called "fixpoint" control-flow position within the basic block (which is what I believe is the intent given the method name...)

Reply:
I seem to have switched the documentation for entry and start?

Reply:
Okay, I've changed this family of methods to use the phrase "dataflow state at fixpoint" when seeking to the dataflow entry point. I've also tried to make it clear that `seek_to_block_{start,end}` are referring to the control flow position, not the dataflow one by explicitly mentioning "first statement" and "terminator". These changes are isolated in the last commit.